recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -285,6 +285,15 @@ this.PreferenceExperiments = {
   },
 
   /**
+   * Get a list of all stored experiment objects.
+   * @resolves {Experiment[]}
+   */
+  async getAll() {
+    const store = await ensureStorage();
+    return Object.values(store.data).map(experiment => Object.assign({}, experiment));
```

Comment:
If we've documented this is supposed to be one-level deep, then I'm happy with the current state, though a comment here saying we're returning a fresh copy to avoid consumers altering our internal state would still be nice. :-)